valhalla/userrequests/duration_utils.py

```diff
@@ -81,7 +81,7 @@ def get_max_ipp_for_userrequest(userrequest_dict):
         duration_hours = duration / 3600.0
         ipp_available = time_allocation.ipp_time_available
         max_ipp_allowable = min((ipp_available / duration_hours) + 1.0, MAX_IPP_LIMIT)
-        max_ipp_allowable = float("{:.3f}".format(max_ipp_allowable))
+        truncated_max_ipp_allowable = ((max_ipp_allowable * 1000.0) // 1) / 1000.0
```

Comment:
I think it would make what you are doing more clear if you used `math.floor` here instead of `// 1`